### DISCUSSION TITLE
How to link the arm64 file?

### DISCUSSION THREAD

**Discussion by woshiccm** [exaloop/codon#204] [Q&A]
If I want to run it on macOS or iOS, how can i link the file? can ld64 do it?

**Comment by arshajii** [comment]
Hi @woshiccm -- `codon` can build an executable directly with `codon build -exe -o <executable_name> program.codon`. Or you can generate an object file with `codon build -obj` and link it with `ld64`, which should also work fine. Let me know if you run into any problems!

**Reply by woshiccm** [reply]
thanks!
